modules/ROOT/pages/salesforce/salesforce-connector-98.adoc

```diff
@@ -0,0 +1,179 @@
+= Salesforce Connector - Get Started - Mule 4
```

Comment:
EDIT INSTRUCTIONS:

Could I get a green only check done today before 1 pm? #481 — Please ignore the connector reference. Thank you.